src/modules/meteor/index.js

```diff
@@ -122,6 +122,12 @@ export function push(api) {
       progressBar: config.enableUploadProgressBar
     });
 
+    list.copy('Pushing Nginx Settings to The Server', {
+      src: config.nginx.settingsPath || resolvePath(__dirname, '../proxy/assets/templates/nginx-default.conf'),
```

Comment:
I think it would be better to use `configuration` or `config` instead of `settings` in the list title and mup config property.